src/userSettings.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @copyright Copyright (c) 2021 Julien Barnoin <julien@barnoin.com>
+ *
+ * @license GNU AGPL version 3 or any later version
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as
+ * published by the Free Software Foundation, either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+import Vue from 'vue'
+import UserSettings from './views/UserSettings'
+
+Vue.prototype.t = t
+Vue.prototype.n = n
+
+export default new Vue({
+	el: '#notifications-user-settings',
+	// eslint-disable-next-line vue/match-component-file-name
```

Comment:
Should fix that instead, but we can take care of it

Reply:
Yeah, I wasn't sure if I should deviate from the naming conventions / capitalization that I saw from the code in another app I used as an example so I copied that as well, I agree it's not great.